app/main.py

```diff
@@ -1,6 +1,24 @@
+import os
+import requests
+
+
+URL = "https://api.weatherapi.com/v1/current.json?"
+CITY = "Paris"
+COUNTRY = "France"
+API_KEY = os.getenv("API_KEY")
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    response = requests.get(URL + f"q={CITY}&key={API_KEY}")
```

Comment:
Consider adding error handling for the API request. This could include checking the response status code and handling exceptions that may occur during the request.